tests/system.rs

```diff
@@ -79,8 +81,21 @@ fn compile_cmdline<T: AsRef<OsStr>>(
 
 const INPUT: &'static str = "test.c";
 const INPUT_ERR: &'static str = "test_err.c";
+const INPUT_MACRO_EXPANSION: &'static str = "test_macro_expansion.c";
+const INPUT_WITH_DEFINE: &'static str = "test_with_define.c";
 const OUTPUT: &'static str = "test.o";
 
+// Copy the source files into the tempdir so we can compile with relative paths, since the commandline winds up in the hash key.
+fn copy_to_tempdir(inputs: Vec<String>, tempdir: &Path)
+{
```

Comment:
This is not common style, please run rustfmt. Also since you don't need to resize `inputs` it may as well be an `&[String]`.


Furthermore, given these are static, you can just do `&[&str]`.

Reply:
Ok, thanks for the hint, I changed that!